packages/app/src/components/DBTracePanel.tsx

```diff
@@ -202,7 +204,14 @@ export default function DBTracePanel({
           <Text size="sm" c="dark.2" my="sm">
             Span Details
           </Text>
-          <RowDataPanel source={traceSourceData} rowId={traceRowWhere} />
+          <RowDataPanel
+            source={
+              traceRowWhere?.type === SourceKind.Log && parentSourceData
+                ? parentSourceData
```

Comment:
I think parentSourceData is a log source only if the user opened a log, but if you open a trace, then the parentSourceData would be a trace type?

Reply:
oh i didn't realize they were two tickets